### DISCUSSION TITLE
Classes from model

### DISCUSSION THREAD

**Discussion by shdxw** [deepjavalibrary/djl#1353] [Q&A] [answered]
Hello everyone! I'm trying to use a pre-trained model (coco) to detect cars and trucks in the photo, I can't select only these classes from the model.  (I don't need the rest of the classes)
How can I specify only the necessary classes from the existing model for the detection?

**Comment by frankfliu** [comment, marked as answer]
@shdxw 
For a pre-trained, it will detect all the classes, the output Shape is fixed (e.g. 1000 classes), what you can do is filter the output and find the class that are are interested in. See this example: https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/inference/PoseEstimation.java#L95
